pkg/planner/core/plan_stats.go

```diff
@@ -38,7 +38,7 @@ func (collectPredicateColumnsPoint) optimize(_ context.Context, plan LogicalPlan
 		return plan, planChanged, nil
 	}
 	predicateNeeded := variable.EnableColumnTracking.Load()
-	syncWait := plan.SCtx().GetSessionVars().StatsLoadSyncWait * time.Millisecond.Nanoseconds()
```

Comment:
This change is not about the max execution time, just wrong time unit is used?

Reply:
Yes, it fix issue #50872.